models/room.js

```diff
@@ -171,7 +171,7 @@ class Room {
       const resp = await dj.send({name: 'requestTrack'});
       track = resp.track;
       track.id = uuid();
-      track.url = `${process.env.BUOY_HTTP_URL}/tracks/${track.id}`;
+      track.url = `${process.env.BUOY_URL}/tracks/${track.id}`;
```

Comment:
yeah so the reason that these are different is that one is for the socket address and one is for the HTTP address. honestly, this is a stupid distinction that I only made because I was rushing to get this out the door, so I think the smarter thing would be to have one unified env variable: `BUOY_HOST`:

```bash
# Get rid of these
BUOY_URL=ws://10.0.13.80:8000/
BUOY_HTTP_URL=http://10.0.13.80:8000

# ...and replace it with this
BUOY_HOST=10.0.13.80
```

which we can then just append `https://` or `ws://` to.

Reply:
ugh actually we probably need another flag, `SSL_ENABLED` which would toggle between `https / wss` and `http / ws` protocols.

Reply:
Updated changes expect .env to define SSL_ENABLED & BUOY_HOST.

PeerServer will then build WebSocket & Http(s) URLs: https://github.com/Jesus/groovebuoy/blob/master/models/peer-server.js#L12-L13